.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ version: 2
     - run:
         name: "Build DEB"
         command: |
-          export DEB_BUILD_OPTIONS=nostrip
+          export DEB_BUILD_OPTIONS=nostrip TARGET_PROFILE=release
```

Comment:
providing this is _supposed_ to be a space between exports, and not a comma.. then I'm good with approving.

Reply:
Yep, just a space.